### PR TITLE
TST: Add Constants For Test Regions

### PR DESCRIPTION
**Pull request by John-P**
I propose defining some constants for WSI regions which contain some tissue for use in the tests rather than putting numbers in each individual test.

To keep the PR simple for initial review and discussion I have just committed the change adding the constants. Of course the existing tests would also have to be updated to use them.

### CODE REVIEW

File: tests/test_wsireader.py [hunk 1]
```diff
@@ -9,6 +9,22 @@
 import cv2
 from click.testing import CliRunner
 
+# -------------------------------------------------------------------------------------
+# Constants
+# -------------------------------------------------------------------------------------
+
+NDPI_TEST_TISSUE_BOUNDS = (30400, 11810, 30912, 12322)
```
Comment (shaneahmed) [inline tests/test_wsireader.py:16]: Looks fine to me, but I cannot see if you are using these in the file somewhere.

**Review by shaneahmed — COMMENTED** [review]
Happy with the proposed changes but I can't see where you are using these.

**Comment by John-P** [comment]
I have adjusted tests to use these region constants.

**Comment by John-P** [comment]
# PR Summary

- Add region constants for WSIReader tests.
- Swap literals in tests for defined constants.

**Pull request merged**